aspnetcore/security/authentication/other-logins.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Short survey of other login providers.
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 11/3/2016
+ms.topic: article
+ms.assetid: BC36CA84-3DE8-496E-9AA2-2F1B74AE8309
+ms.prod: aspnet-core
+uid: security/authentication/otherlogins
+---
+# Short survey of other login providers
+
+<a name=security-authentication-other-logins></a>
+
+Here are setup instructions for some other common OAuth login providers. Third-party NuGet packages such as the ones maintained by [aspnet-contrib](https://www.nuget.org/packages?q=owners%3Aaspnet-contrib+title%3AOAuth) can be used to complement login providers implemented by the ASP.NET Core team.
```

Comment:
in general, "sign in" is preferred over "log in"  in this context maybe "authentication providers" is better